Lib/typing.py

```diff
@@ -964,7 +964,7 @@ def __getattr__(self, attr):
             return self._name or self.__origin__.__name__
 
         # We are careful for copy and pickle.
-        # Also for simplicity we just don't relay all dunder names
+        # Also for simplicity we don't relay any dunder names
```

Comment:
This comment was something I tripped over while researching bpo-46581. I don't understand what the original means. I think the above captures the sentiment more clearly. I'm happy to remove this edit, if I missed the mark.

Reply:
Sorry, do you mind splitting out the copyedits into another PR please? I plan to backport them to 3.9 and that will make that easier. You don't have to create a new bpo issue just for that, just a normal PR will do. Thank you!

Reply:
Done: #31684.